### PR TITLE
Fix untranslated module names

### LINKED ISSUE #3016: NUX Landing: module description is not translated
screen shot 2015-11-16 at 10 08 12 am

https://github.com/Automattic/jetpack/blob/3.8.0/views/admin/landing-page-templates.php#L62

From `jetpackL10n `:

```
{"name":"Photon","description":"Speed up images and photos.","jumpstart_desc":"Mirrors and serves your images from our free and fast i</strong, improving your site\u2019s performance with no additional load on your servers.","sort":25,"recommendation_order":1,"introduced":"2.0","changed":"","deactivate":true,"free":true,"requires_connection":true,"auto_activate":"No","module_tags":["Photos et vid\u00e9os","Apparence","Recommand\u00e9 "],"feature":["Recommended","Jumpstart","Performance-Security"],"additional_search_queries":"photon, image, cdn, performance, speed","module":"photon","activated":true,"deactivate_nonce":"5c0a4a2e0d","activate_nonce":"5daf0055ca","available":true,"short_description":"Speed up images and photos.","configure_url":"https:\/\/jeherve.com\/wp-admin\/admin.php?page=jetpack&configure=photon","learn_more_button":"<a class=\"button-secondary more-info-link\" href=\"#\">Plus d&rsquo;info.<\/a>","long_description":"\t<p>Donnez un coup de Boost \u00e0 votre site en chargeant vos images depuis le CDN (Content Delivery Network) de WordPress.com. Nous cachons vos images et les servont depuis nos propres serveurs, ce qui r\u00e9duit d&rsquo;autant la charge sur votre h\u00e9bergement, en un seul clic.<\/p>\n","search_terms":"photon, image, cdn, performance, speed","configurable":false},
```

It looks like some parameters (`long_description`, `learn_more_button`) are translated, but not `short_description`.

### PR DESCRIPTION
**Pull request by eliorivero**
fixes #3016 #3102

To test this PR, run `grunt shell` or at least `php tools/build-module-headings-translations.php`

### CODE REVIEW

**Comment by eliorivero** [comment]
@zinigor based on what we talked yesterday, I've implemented a lookup table for the translations and eliminated those variables inside `_x`.
This fixes the bugs where modules were not displaying its translated strings. Would love to know what you think.

cc @jeherve as well

File: jetpack.php [hunk 1]
```diff
@@ -51,6 +51,7 @@
 require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-tracks.php'        );
 
 if ( is_admin() ) {
+	require_once( JETPACK__PLUGIN_DIR . '/modules/module-headings.php');
```
Comment (zinigor) [inline jetpack.php]: Although it's a minor thing, let's not use a slash before `modules` here?

**Comment by zinigor** [comment]
This is awesome, thank you! The only thing that I have changed is that I made the file be included on every run, not on admin runs only. This is in order to fix grunt.

**Comment by jeherve** [comment]
This causes a Fatal Error on my test site until I run `grunt shell`. Should we commit the grunt changes to master as well, to avoid issues for folks running `master` on production sites?

```
[11-Feb-2016 14:33:48 UTC] PHP Fatal error:  Call to undefined function jetpack_get_module_i18n_tag() in wp-content/plugins/jetpack/class.jetpack.php on line 2548
```

**Comment by dereksmart** [comment]
@jeherve yes typically this (and merging into master-stable) should be done after every merge session.  

I have updated both master and master-stable with updated headers from grunt

**Pull request merged**